docs/topics/multiplatform/multiplatform-compatibility-guide.md

```diff
@@ -192,4 +192,47 @@ The available scopes are `Api`, `Implementation`, `CompileOnly`, and `RuntimeOnl
 
 In Kotlin 1.8.0, an error is introduced when using old configuration names in hard-coded strings.
 
-For more information, see the [corresponding issue in YouTrack](https://youtrack.jetbrains.com/issue/KT-35916/).
\ No newline at end of file
+For more information, see the [corresponding issue in YouTrack](https://youtrack.jetbrains.com/issue/KT-35916/).
+
+## Gradle properties for hierarchical support
```

Comment:
How about ensuring that the title describes changes briefly? Like, `Removal of Gradle properties for hierarchical support` or `Deprecation of Gradle Properties for hierarchical support`. Currently it's not clear at a glance what this section is about: were those properties added? renamed? behavior changed?

Reply:
`Hierarchical support` isn't very well-established term, unfortunately, and because of that it sounds awkward ("Hierarchical" what? Is the support hierarchical?) I'd suggest using more verbose, but unambiguous form `Hierarchical project structure support` here and below

Reply:
Thank you for the suggestions! I made the header a little shorter, but fixed the term below and added a link to the corresponding page